pandas_gbq/gbq.py

```diff
@@ -956,63 +931,79 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
     """
 
+    def _create_df(rows, columns, schema, index_col, col_order):
```

Comment:
Once https://github.com/GoogleCloudPlatform/google-cloud-python/pull/4354 goes in (adds `to_dataframe()` method to the `google-cloud-bigquery` library), the `_create_df` function may be unnecessary.

Reply:
Sounds good, look forward to it!